packages/amplication-client/src/Project/NewProject.tsx

```diff
@@ -56,31 +58,30 @@ type Props = {
 const NewProject = ({ onProjectCreated }: Props) => {
   const { onNewProjectCompleted } = useContext(AppContext);
   const { trackEvent } = useTracking();
-  const [createProject, { error, loading }] = useMutation<DType>(
-    CREATE_PROJECT,
-    {
-      onCompleted: (data) => {
-        trackEvent({
-          eventName: AnalyticsEventNames.ProjectCreate,
-          projectName: data.createProject.name,
-        });
-        onProjectCreated();
-        onNewProjectCompleted(data.createProject);
-      },
-    }
-  );
+  const [createProject, { loading }] = useMutation<DType>(CREATE_PROJECT, {
+    onCompleted: (data) => {
+      trackEvent({
+        eventName: AnalyticsEventNames.ProjectCreate,
+        projectName: data.createProject.name,
+      });
+      onProjectCreated();
+      onNewProjectCompleted(data.createProject);
+    },
+  });
 
   const handleSubmit = useCallback(
-    (data: CreateProjectType) => {
+    (data: CreateProjectType, { setErrors }) => {
       createProject({
         variables: {
           data,
         },
-      }).catch(console.error);
+      }).catch((err) => {
+        setErrors({ name: ERROR });
```

Comment:
Please keep the original error text. The current implementation will display the same error message for all types of errors that can be returned from the server which is not correct 
```suggestion
        setErrors({ name: formatError(err); });
```